liveclasses/models.py

```
@@ -0,0 +1,64 @@
+from .jass_jwt import JaaSJwtBuilder
+
+import uuid
+import os
+
+
+class LiveClass(models.Model):
+    STATUS_CHOICES = [
+        ("Upcoming", "Upcoming"),
+        ("Ongoing", "Ongoing"),
+        ("Completed", "Completed"),
+    ]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)